models/app.model.js

```diff
@@ -13,7 +13,23 @@ exports.readArticleByID = (id) => {
 
   return db
     .query(
-      "SELECT articles.author, articles.title, articles.article_id, articles.topic, articles.created_at, articles.votes, articles.article_img_url FROM articles WHERE article_id = $1",
+      `SELECT 
+  articles.author, 
+  articles.title, 
+  articles.article_id, 
+  articles.topic, 
+  articles.created_at, 
+  articles.votes, 
+  articles.article_img_url, 
+  COUNT(comments.comment_id) AS comment_count
```

Comment:
Nice, you can have a look at casting to make this comment_count a number rather than a string 👀